### DISCUSSION TITLE
第4节课笔记(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#315] [笔记]
请同学们把第四节课笔记交到这里哦~

**Comment by zhjunqin** [comment]
zhjunqin: https://blog.csdn.net/zhjunqin/article/details/135572684?spm=1001.2014.3001.5502

**Comment by onlyzer0** [comment]
onlyzer0：https://blog.csdn.net/qq_42907812/article/details/135574702?spm=1001.2014.3001.5502

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/f5115d59085243ca89cbbc238878c9bf?pvs=4
